src/qt/guiutil.cpp

```diff
@@ -212,11 +212,14 @@ bool parseBitcoinURI(QString uri, SendCoinsRecipient *out)
     // Convert bitcoin:// to bitcoin:
     //
     //    Cannot handle this later, because bitcoin:// will cause Qt to see the part after // as host,
-    //    which will lower-case it (and thus invalidate the address).
+    //    which will lower-case it (and thus invalidate the address). Workaround only for Qt4 support.
+
+#if QT_VERSION < 0x050000
```

Comment:
IMO just drop these lines of code, so that it gets to the error message for Qt4 too. This makes behavior under both consistent. Also: Qt4 support is under-tested and going to be removed in the not-so-far future, so might be better to not have a special, not strictly necessary workaround for it.